source/agora/test/EnrollDifferentUTXO.d

```diff
@@ -112,7 +113,7 @@ unittest
         .array;
 
     // 8 utxos for freezing, 24 utxos for creating a block later
-    txs ~= spendable[4].split(WK.Keys.NODE2.address.repeat(8)).sign();
+    txs ~= spendable[4].split(genesis_validator_keys[0].address.repeat(8)).sign();
```

Comment:
Can you explain why this should be changed again? I can not remember the reason.

Reply:
I will extract this change into a separate commit with an explanation of the change. But it is basically because the test is using first node which is first enrollment in Genesis. NODE2 is not always the first node as the enrollments are ordered by utxo.